app/src/main/java/com/kickstarter/ui/compose/designsystem/KSButtons.kt

```diff
@@ -732,10 +733,13 @@ fun KSButton(
     textStyle: TextStyle = typographyV2.body,
     textColor: Color,
     shape: RoundedCornerShape? = null,
+    shouldWrapContentWidth: Boolean = false
 ) {
+    val mod = if (shouldWrapContentWidth)
+        modifier.wrapContentWidth()
+    else modifier.fillMaxWidth()
```

Comment:
Quite probably for our design system health we should not set width for buttons within the button itself, but on the modifier outside of the button itself, (fyi: modifiers in compose do follow the builder pattern, if we setup with within the button itself it will be the latest one applied)
Will add a follow up design system refactor ticket for this specific point to not set  `fillMaxWidth()` nor `wrapContentWidth()` within the button component. Adding this solution temporarily to not hold any release